randovania/games/super_metroid/json_data/Brinstar.txt

```diff
@@ -817,13 +839,25 @@ Extra - asset_id: 96
   * Layers: default
   * Normal Door to Blue Brinstar Energy Tank Room/Door to Blue Brinstar Boulder Room
   > Door to Blue Brinstar Double Missile Room
-      Gravity Suit
+      All of the following:
+          Any of the following:
+              # Get out of the water
+              Gravity Suit or Movement (Intermediate)
```

Comment:
I'd add knowledge as an option as well - one trick is to just jump in the door transition. This skips falling in the water altogether and requires no real skill besides knowing that you're able to do it.

Reply:
i thought that was what the movement stood for?

Reply:
The movement is pretty trivial, it really shouldn't be intermediate.